kong/plugins/request-transformer/access.lua

```diff
@@ -238,15 +275,26 @@ local function transform_url_encoded_body(conf, body, content_length)
     end
   end
 
-  if removed or replaced or added or appended then 
+  if renamed or removed or replaced or added or appended then
     return true, encode_args(parameters)
   end
 end
 
 local function transform_multipart_body(conf, body, content_length, content_type_value)
-  local removed, replaced, added, appended = false, false, false, false
+  local renamed, removed, replaced, added, appended = false, false, false, false, false
   local parameters = multipart(body and body or "", content_type_value)
 
+  if content_length > 0 and #conf.rename.body > 0 then
+    for _, old_name, new_name in iter(conf.rename.body) do
+      if parameters:get(old_name) then
+        local value = parameters:get(old_name)
```

Comment:
Apparently this call returns a table:
```
{
  headers = { 'Content-Disposition: form-data; name="originalparam"' },
  name = "originalparam",
  value = "yes"
}, client: 127.0.0.1, server: kong, request: "POST /request HTTP/1.1", host: "test7.com"
```

You'd have to make sure you are retrieving the `value` field of that table. I do not know lua-multipart and I don't know if the API is made to return such a table or the actual underlying `value` field. You'll have to investigate if you are using it properly.